### ISSUE TITLE
Accessibility: breaks page when using screenreader

### ISSUE THREAD

**Issue by callumacrae** [monterail/vue-multiselect#156]
Hello!

Currently, when using the page with a screenreader, the multiselect component creates a loop that the screenreader can't get out of, breaking any page the component is used on.

The best way to do this would be to hide the multiselect component for screenreaders and use `<select multiple>` instead, but looking at the generated output that doesn't look like it would be that easy.

Have you got any thoughts on this?

**Comment by callumacrae** [comment]
Oh yeah, the reason it doesn't work: when the screenreader goes to read the first element it causes the select element to be defocused, so it closes.

Following gif is of me repeatedly pressing VO+right in VoiceOver, which should go through the page reading things, not get stuck in a loop:

![](https://cloud.githubusercontent.com/assets/472830/20833097/b50167d4-b885-11e6-8d64-478b304111a8.gif)

The text in the black bubble is what the screen reader is reading out.

**Comment by shentao** [comment]
That might be hard. Thanks for bringing this up! Accessibility is really important for me so I hope I can find a way to fix this!

**Comment by NiltiakSivad** [comment]
Would really like this as well!

**Comment by jbbergy** [comment]
Hi,

some news about the accessibility ? the screenreader read the options as empty
